src/components/pages/auth/logout.tsx

```diff
@@ -31,18 +31,25 @@ export function Logout({ oauth }: { oauth?: boolean }) {
   useEffect(() => {
     checkInstance({ redirect: false })
 
-    const redirection = filterUnwantedRedirection({
-      desiredPath: sessionStorage.getItem('previousPathname'),
-      unwantedPaths: [settingsUrl, loginUrl, registrationUrl],
-    })
+    const cookieSession = AuthSessionCookie.parse()
+    const isSSO = cookieSession?.authentication_methods
```

Comment:
We can use `io-ts` here (can also be a follow up :smile: )

Reply:
Just for line 36?
I'd only use it if it's as short tbh :)